src/test/java/andreeaG/tests/tema8/pageTests/SimpleDynamicButtonsTest.java

```diff
@@ -0,0 +1,23 @@
+package andreeaG.tests.tema8.pageTests;
+
+import andreeaG.tests.tema8.pageObjects.SimpleDynamicButtonsPage;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class SimpleDynamicButtonsTest extends BaseTest {
+
+    @Test(groups = {"mobile","desktop"})
+    public void verifyClickOnAllButtons() {
+        driver =new ChromeDriver();
```

Comment:
Remove this line. We have BeforeMethods in BaseTest that create the driver instances.